### ISSUE TITLE
New Documentation - Need Code Table Documentation

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/documentation-wiki#245]
Please complete the form to the best of your ability. This will help us find the document and complete the update.

ArctosDB.org url: suggest - https://handbook.arctosdb.org/documentaion/code_tables.html

Github code: suggest - https://github.com/ArctosDB/documentation-wiki/blob/gh-pages/_documentation/Code_Tables.markdown

The following need to be added/changed/deleted 

 - [ ] We need something that explains in general what code tables are, how they are used, and who manages them as part of Documentation

Note that the documents above do not yet exist, but I am requesting this to complete https://arctos.database.museum/Admin/user_roles.cfm

**No need to complete below this point - these are reminders for the person working on the update**

 - [ ] Add a link to this issue at the top of the document (use "Add to current update request" as the text for the link)
 - [ ] Add a link to this issue in the Community discussion section of the document (use "Github Issue #X" as the text for the link.)

When the above are complete:

 - [ ] Comment the link created at the top of the document. (use  [//]: # before the link so it will not display on the page, but will remain in the code)
- [ ] Remove the under construction bear banner if it is there

Notify the Arctos Community of the update:

 - [ ] Social Media
 - [ ] Google Group
 - [ ] Newsletter

**Comment by dustymc** [comment]
This exists as https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html

**Comment by Jegelewicz** [comment]
Does it though? I don't feel like we define WHAT code tables are and how they are used in Arctos? Maybe that How To is good enough, but I am looking at required reading for new code table admins and do we think that How To is enough?

**Comment by dustymc** [comment]
Docs can always be better, but lets don't make more - merge, move (but leave redirects), whatever, I just don't want one more THING to fail to find.

**Comment by Jegelewicz** [comment]
> one more THING to fail to find.

Trying to avoid that by putting stuff in https://arctos.database.museum/Admin/user_roles.cfm but also, as long as it is in the handbook - we should be able to find it?

**Comment by dustymc** [comment]
"Denormalization is bad" has no boundaries.

**Comment by Jegelewicz** [comment]
Our oldest open issue in documentation_wiki - https://github.com/ArctosDB/documentation-wiki/issues/123 got me started down this path. Looking at user roles and how well we explain them to incoming collections seems to be a priority for me! Pretty sure a lot of people have access to some of these roles who have never read the documentation or a how to...

**Comment by Jegelewicz** [comment]
> "Denormalization is bad" has no boundaries.

? I'm trying to put everything in the handbook - all that table has is links....

**Comment by dustymc** [comment]
>how well we explain

Agreed! Part of that is being able to find things, and the easiest path to that has only one THING to find. Documentation is increasingly scattered; I'm not suggesting anything at all about the content, only the structure. Nobody can be expected to dig through a dozen pages to find something, and search doesn't work well across lots of pages. I don't like this sort of thing:

<img width="406" alt="Screen Shot 2021-10-06 at 10 55 16 AM" src="https://user-images.githubusercontent.com/5720791/136257555-e70ded44-8d17-4fb2-9177-2df444ae8c6a.png">

<img width="365" alt="Screen Shot 2021-10-06 at 10 55 43 AM" src="https://user-images.githubusercontent.com/5720791/136257566-0d65013c-b3e4-47dd-bfc3-0a0527f722ce.png">

(although some - hopefully most! - of that's probably just "go see ``locality`` for more information").

I'm just asking for better organization, which I think generally means fewer pages, and I think you're proposing to create more pages.

>people have access to some of these roles who have never read the documentation

I still kinda like Gordon's idea of a test....

**Comment by Jegelewicz** [comment]
Part of that is due to people's short attention span. We can't put everything about "Locality" in one document and expect people to wade through it. In the same way we have created short video tutorial, we have also created short bits of "How-To" for very specific tasks. I am not saying one or the other method is better or worse and it probably depends on how you see the world.

**Comment by dustymc** [comment]
https://handbook.arctosdb.org/documentation/authorities.html
